pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.black]
+line-length=120
+target-version=["py38"]
+
+[tool.isort]
```

Comment:
@ZhukovGreen I noticed this in the doc

https://pycqa.github.io/isort/docs/configuration/black_compatibility/

https://pycqa.github.io/isort/docs/configuration/profiles/

From that we're missing this, should we add that or do the black/isort not collide with each other with current config?
```
force_grid_wrap: 0
ensure_newline_before_comments: true
```

Reply:
They are not collide in this particular setup. Time will show, however in my projects I never seen the need to solve the conflict. Time will show.

Reply:
I like Eric's suggestion. It would make the config a bit more concise as well:
```
atomic=true
line_length = 120
lines_after_imports=2
lines_between_types=1
use_parentheses = true
balanced_wrapping = true
include_trailing_comma = true
multi_line_output = 3
```
down to:
```
profile = "black"
atomic = true
line_length = 120
lines_after_imports = 2
lines_between_types = 1
balanced_wrapping = true
```

Reply:
OK, should be addressed by #279